application/ui/components/endpoint.jsx

```diff
@@ -312,6 +311,12 @@ module.exports = React.createClass({
         }
     },
 
+    changeHandler : function(value)
+    {
+        console.log(value);
```

Comment:
it's actually pretty helpful IMO.  makes it easy to see the structure of the request body before you actually try it.